modules/databend/examples_test.go

```diff
@@ -0,0 +1,89 @@
+package databend_test
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"log"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/databend"
+)
+
+func ExampleRun() {
+	ctx := context.Background()
+
+	databendContainer, err := databend.Run(ctx,
+		"datafuselabs/databend:v1.2.615",
+		databend.WithUsername("databend"),
+		databend.WithPassword("databend"),
+	)
+	defer func() {
+		if err := testcontainers.TerminateContainer(databendContainer); err != nil {
+			log.Printf("failed to terminate container: %s", err)
+		}
+	}()
+	if err != nil {
+		log.Printf("failed to start container: %s", err)
+		return
+	}
+	// }
+
+	state, err := databendContainer.State(ctx)
+	if err != nil {
+		log.Printf("failed to get container state: %s", err)
+		return
+	}
+
+	fmt.Println(state.Running)
+}
+
+func ExampleRun_connect() {
+	ctx := context.Background()
+
+	databendContainer, err := databend.Run(ctx,
+		"datafuselabs/databend:v1.2.615",
+		databend.WithUsername("root"),
+		databend.WithPassword("password"),
+	)
+	defer func() {
+		if err := testcontainers.TerminateContainer(databendContainer); err != nil {
+			log.Printf("failed to terminate container: %s", err)
+		}
+	}()
+	if err != nil {
+		log.Printf("failed to start container: %s", err)
+		return
+	}
+
+	connectionString, err := databendContainer.ConnectionString(ctx)
+	if err != nil {
+		log.Printf("failed to get connection string: %s", err)
+		return
+	}
+
+	db, err := sql.Open("databend", connectionString)
+	if err != nil {
+		log.Printf("failed to connect to Databend: %s", err)
+		return
+	}
+	defer db.Close()
+
+	if err = db.Ping(); err != nil {
+		log.Printf("failed to ping Databend: %s", err)
+		return
+	}
+	var i int
+	row, err := db.Query("select 1")
+	if err != nil {
+		log.Printf("failed to query: %s", err)
+		return
+	}
+	err = row.Scan(&i)
+	if err != nil {
+		log.Printf("failed to scan row: %s", err)
+		return
+	}
+
+	fmt.Println(i)
```

Comment:
bug: Missing Output definition for validation of the example.

Reply:
Thanks, fixed.